src/main/java/net/minecraftforge/common/ForgeHooks.java

```diff
@@ -1326,4 +1327,11 @@ public static void onEntityEnterSection(Entity entity, long packedOldPos, long p
         MinecraftForge.EVENT_BUS.post(new EntityEvent.EnteringSection(entity, packedOldPos, packedNewPos));
     }
 
+    public static float getDamageBonus(Player player, ItemStack weapon, Entity target, float enchantmentBonus)
```

Comment:
I would probably reduce the scope type to `LivingEntity` for reasons I'll explain in the main comment.